openstack/cinder/templates/etc/_cinder-policy.yaml.tpl

```diff
@@ -0,0 +1,143 @@
+# Policy yaml file generated with
+# tox -egenpolicy
+# All defaults are supplied in the
+# generated policy file.  Only diffs from
+# defaults need to be specified here
+
+"context_is_cloud_admin":  "role:cloud_volume_admin"
+"context_is_quota_admin":  "role:resource_service"
+"context_is_admin":  "rule:context_is_cloud_admin"
+"owner": "project_id:%(project_id)s"
+"member": "role:member and rule:owner"
+"viewer": "role:volume_viewer and rule:owner"
+"admin": "role:volume_admin and rule:owner"
+"context_is_volume_admin": "rule:context_is_admin or rule:admin"
+"context_is_editor": "rule:context_is_volume_admin or rule:member"
+"context_is_viewer":  "rule:context_is_editor or rule:viewer"
+"view_all": "role:member or role:volume_viewer or role:volume_admin or role:cloud_volume_admin"
+"edit_all": "role:member or role:volume_admin or role:cloud_volume_admin"
+"default": "rule:context_is_admin"
```

Comment:
Release-Notes state, that `default` is not used inside the code anymore, if I read this correctly.